src/App.tsx

```diff
@@ -1,24 +1,156 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import * as todoServise from './api/todos';
+import { Footer } from './components/Footer';
+import { TodoList } from './components/TodoList';
+import { Status } from './types/Status';
+import { Todo } from './types/Todo';
+import { prepareTodos } from './utils/filterTodos';
 import { UserWarning } from './UserWarning';
+import { TodoError } from './components/TodoError';
+import { Error } from './types/Error';
 
-const USER_ID = 0;
+const USER_ID = 6909;
 
 export const App: React.FC = () => {
+  const [todoTitle, setTodoTitle] = useState('');
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [sortField, setSortField] = useState(Status.All);
+  const [error, setError] = useState(Error.NONE);
+  const [isLoading, setIsLoading] = useState(false);
+  const [isActiveIds, setIsActiveIds] = useState<number[]>([]);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+
+  useEffect(() => {
+    todoServise.getTodos(USER_ID)
+      .then(setTodos)
+      .catch(() => setError(Error.LOAD));
+  }, []);
+
   if (!USER_ID) {
     return <UserWarning />;
   }
 
+  const todosExist = todos.length > 0;
+  const completedTodos = todos.filter(todo => todo.completed);
+  const notCompletedTodos = todos.filter(todo => !todo.completed);
```

Comment:
get this values only when you need to delete, you can create separated functions for getting todos(completed or not)